JavaAssignment/src/soumya/Bank.java

```diff
@@ -0,0 +1,100 @@
+package soumya;
+
+import java.util.Scanner;
+
+public class Bank {
+	long account_no;
+	int choice;
+	double amount;
+	double amountDep, amountWithdraw;
+	String name;
+	Scanner sc = new Scanner(System.in);
+
+	// Method for getting input from user
+	void inputOutput() {
```

Comment:
try to write parameterized method so other can use your method.
and take input from the user in main method.